kustomization/base/configMap.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: app-config
+  namespace: my-app-namespace
+data:
+  API_GATEWAY_URI: "http://172.29.25.105:32000"

```

Comment:
why is this hardcoded?

Reply:
Hardcoded the Node Ip and Node port of minikube for intial setup.

Reply:
https://github.com/imzainazm/microservices-chat-demo/pull/6#discussion_r1652433474

> Could you explain why do you need to do this?

Reply:
One of the microservice needed this url of the other microservice to connect with.

Reply:
yes it is suitable with service-name : port (k8s internal server DNS feature)

Reply:
Please update it.

Reply:
Updated now.